test_data/asm/permutations/block_to_block.asm

```diff
@@ -43,9 +43,7 @@ machine Main with degree: 256 {
     Binary4 bin;
     Binary4x bin4;
 
-    // two permutations to machine bin
     instr or X, Y -> Z link ~> Z = bin.or(X, Y);
-    instr or_into_B X, Y link ~> B' = bin.or(X, Y);
```

Comment:
unfortunately Halo2 has a bug somewhere and we can't do this.
This is testes in other places (including RISCV tests) with witgen

Reply:
is it possible to disable halo2 for the test instead?

Reply:
we still want to run halo2 proofs for these tests to test permutations between machines

Reply:
But do we test the previous functionality somewhere else?

Reply:
language/compiler/witgen wise yes, the RISCV machine has this stuff and more